### PR TITLE
Drop Exceptions

### LINKED ISSUE #73: Drop Exceptions
While refactoring things for #56 I was trying for a brief time to unify subscriptions to `ActiveSupport` instrumentation and `HijackRenderException` and I started to wonder: Why do we do full blown exception tracking with this gem in a time series database? There are plenty of services, even FOSS ones, that do this ([errbit](https://github.com/errbit/errbit) || [sentry](https://sentry.io)) in a much nicer way.

I can understand that we maybe want to measure exceptions occurrence + some tags for it by default. But full blown?

What do you think?

### PR DESCRIPTION
**Pull request by hennevogel**
Fixes #73

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -303,12 +275,6 @@ and (on final error) log an error and discard the values.
 
 The data points are simply discarded.
 
-### What happens, when the InfluxDB client or this gem throws an exception? Will the user see 500 errors?
```
Comment (ChrisBr) [inline README.md:312]: That's still valid? At least that you don't see an exception? I would still log them tho.

File: lib/influxdb-rails.rb [hunk 2]
```diff
@@ -31,7 +25,6 @@ def configure
 
         yield configuration
         self.client = nil # if we change configuration, reload the client
-        InfluxDB::Logging.logger = configuration.logger unless configuration.logger.nil?
```
Comment (ChrisBr) [inline lib/influxdb-rails.rb:34]: Would still log them, no? Otherwise it is quite hard to debug ...

**Review by ChrisBr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hennevogel) [reply README.md:312]: No it's not, this was part of the exception code. See the dropped `.report_exception` from `lib/influxdb-rails.rb`

File: lib/influxdb-rails.rb (hunk 2 above)
Comment (hennevogel) [reply lib/influxdb-rails.rb:34]: Yes we still log but just using `Rails.logger` directly...

File: README.md (hunk 1 above)
Comment (ChrisBr) [reply README.md:312]: Huh? We still wrap call into a `rescue` e.g. https://github.com/influxdata/influxdb-rails/blob/master/lib/influxdb/rails/middleware/simple_subscriber.rb#L18

Comment (hennevogel) [reply README.md:312]: Added it again.

File: lib/influxdb/rails/middleware/request_subscriber.rb [hunk 3]
```diff
@@ -13,7 +13,7 @@ def call(_name, start, finish, _id, payload)
             tags:      tags(payload),
             timestamp: timestamp(finish)
         rescue StandardError => e
-          log :error, "[InfluxDB::Rails] Unable to write points: #{e.message}"
+          ::Rails.logger.error("[InfluxDB::Rails] Unable to write points: #{e.message}")
```
Comment (ChrisBr) [inline lib/influxdb/rails/middleware/request_subscriber.rb:16]: Is the escaping with `::` necessary?

**Review by ChrisBr — APPROVED** [review]
If you fix the specs 😉

File: lib/influxdb/rails/middleware/request_subscriber.rb (hunk 3 above)
Comment (hennevogel) [reply lib/influxdb/rails/middleware/request_subscriber.rb:16]: If you want this to be `Rails` and not `InfluxDB::Rails` I guess so...

**Comment by hennevogel** [comment]
Here goes nothing...

**Pull request merged**